lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -156,17 +174,14 @@ def reject_all_values_if_none(source_list)
     # e.g. *.github.com asdf.github.com becomes *.github.com
     def dedup_source_list(sources)
       sources = sources.uniq
-      wild_sources = sources.select { |source| source =~ STAR_REGEXP }
+      host_source_expressions = sources.map { |source| parse_source_expression(source) }
+      # TODO: Split by source expression type.
```

Comment:
Is this TODO before the PR merges? If not we should likely document it in a ticket and remove this comment.